lib/src/main/java/io/ably/lib/rest/Auth.java

```diff
@@ -324,7 +325,17 @@ public JsonObject asJSON() {
 	 * @param callback (err, tokenDetails)
 	 */
 	public TokenDetails authorise(AuthOptions options, TokenParams params) throws AblyException {
-		return tokenAuth.authorise(options, params);
+		TokenDetails tokenDetails = tokenAuth.authorise(options, params);
+
+		/* RTC8
+		 *  If authorise is called with AuthOptions#force set to true
+		 *  the client will obtain a new token, disconnect the current transport
+		 *  and resume the connection
+		 */
+		if (options != null && options.force && ably instanceof AblyRealtime) {
```

Comment:
Does this impact our ability to compile a REST-only library? I know we don't do that right now but I don't really like the pollution.

I'm not sure what the cleanest way is to do this, but at a minimum we could just do `ably.getClass().getName().equals( ....) `